addons/l10n_account_edi_ubl_cii_tests/tests/test_xml_cii_fr.py

```diff
@@ -0,0 +1,272 @@
+
+        cls.partner_1 = cls.env['res.partner'].create({
+            'name': "partner_1",
+            'street': "Rue Jean Jaurès, 42",
+            'zip': "75000",
+            'city': "Paris",
+            'vat': 'FR05677404089',
+            'country_id': cls.env.ref('base.fr').id,
+            'bank_ids': [(0, 0, {'acc_number': 'FR15001559627230'})],
+            'phone': '+1 (650) 555-0111',
+            'email': "partner1@yourcompany.com",
+            'ref': 'seller_ref',
+        })
+
+        cls.partner_2 = cls.env['res.partner'].create({
+            'name': "partner_2",
+            'street': "Rue Charles de Gaulle",
+            'zip': "52330",
+            'city': "Colombey-les-Deux-Églises",
+            'vat': 'FR35562153452',
+            'country_id': cls.env.ref('base.fr').id,
+            'bank_ids': [(0, 0, {'acc_number': 'FR90735788866632'})],
+            'ref': 'buyer_ref',
+        })
+
+        cls.tax_21 = cls.env['account.tax'].create({
+            'name': 'tax_21',
+            'amount_type': 'percent',
+            'amount': 21,
+            'type_tax_use': 'sale',
+            'country_id': cls.env.ref('base.fr').id,
+        })
+
+        # remove this tax, otherwise, at import, this tax with children taxes is selected and the total is wrong
+        cls.tax_armageddon.children_tax_ids.unlink()
+        cls.tax_armageddon.unlink()
+
+        cls.tax_20 = cls.env['account.tax'].create({
+            'name': 'tax_20',
+            'amount_type': 'percent',
+            'amount': 20,
+            'type_tax_use': 'sale',
+            'country_id': cls.env.ref('base.fr').id,
+        })
+
+        cls.tax_12 = cls.env['account.tax'].create({
+            'name': 'tax_12',
+            'amount_type': 'percent',
+            'amount': 12,
+            'type_tax_use': 'sale',
+            'country_id': cls.env.ref('base.fr').id,
+        })
+
+        cls.tax_55 = cls.env['account.tax'].create({
+            'name': 'tax_55',
+            'amount_type': 'percent',
+            'amount': 5.5,
+            'type_tax_use': 'sale',
+            'country_id': cls.env.ref('base.fr').id,
+        })
+
+        cls.tax_10 = cls.env['account.tax'].create({
+            'name': 'tax_10',
+            'amount_type': 'percent',
+            'amount': 10,
+            'type_tax_use': 'sale',
+            'country_id': cls.env.ref('base.fr').id,
+        })
+
+        cls.tax_0 = cls.env['account.tax'].create({
+            'name': 'tax_0',
+            'amount_type': 'percent',
+            'amount': 0,
+            'type_tax_use': 'sale',
+            'country_id': cls.env.ref('base.fr').id,
+        })
+
+        cls.acc_bank = cls.env['res.partner.bank'].create({
+            'acc_number': 'FR15001559627231',
+            'partner_id': cls.company_data['company'].partner_id.id,
+        })
+
+        cls.invoice = cls.env['account.move'].create({
```

Comment:
since this invoice is only use in test_export_pdf, I think it's better to create it directly in this test method instead of the setupclass